docs/src/main/asciidoc/logging-guide.adoc

```diff
@@ -37,6 +37,10 @@ Logging to a file is also supported but not enabled by default.  To configure or
 |quarkus.log.file.format|%d{yyyy-MM-dd HH:mm:ss,SSS} %h %N[%i] %-5p [%c{3.}] (%t) %s%e%n|The format pattern to use for logging to a file; see <<format_string>>.
 |quarkus.log.file.level|ALL|The minimum log level to write to the log file.
 |quarkus.log.file.path|quarkus.log|The path of the log file.
+|quarkus.log.file.rotation.max-file-size||The maximum file size of the log file after which a rotation is executed.
+|quarkus.log.file.rotation.max-backup-index|1| The maximum number of backups to keep.
+|quarkus.log.file.rotation.file-suffix|-yyyy-MM-dd| Rotating log file suffix. The format of suffix value has to be understood by `java.text.SimpleDateFormat`.
```

Comment:
Where does this default value come from?

Reply:
Good catch. This part was not attended after the previous round of reviews. Indeed there is no default value. I am addressing it now.

Reply:
All good now. Except for the failing Windows Build related to #2363 / #2187